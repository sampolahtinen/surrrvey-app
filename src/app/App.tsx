import React from "react";
import { Switch, Route } from "react-router";
import LandingPage from "../pages/LandingPage/LandingPage";
import SurveysPage from "../pages/SurveyPage";
import SurveyDetailsPage from "../pages/SurveysDetailsPage";

const App: React.FC = () => {
  return (
    <Switch>
      <Route exact path="/" component={LandingPage} />
      <Route exact path="/surveys" component={SurveysPage} />
      <Route path="/surveys/:id" component={SurveyDetailsPage} />
    </Switch>
  );
};

export default App;
