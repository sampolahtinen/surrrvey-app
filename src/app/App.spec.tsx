import React from "react";
import { shallow } from "enzyme";
import App from "./App";

describe("App Component", () => {
  it("renders properly", () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
