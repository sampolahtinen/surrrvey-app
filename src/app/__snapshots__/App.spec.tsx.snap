// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Component renders properly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/surveys"
  />
  <Route
    component={[Function]}
    path="/surveys/:id"
  />
</Switch>
`;
